packages/dashboard/src/app/serializers/templates.js

```diff
@@ -36,7 +32,6 @@ export default function reshapeTemplateObject(
     ...originalTemplateData,
     id,
     slug,
-    centerTargetAction: `${APP_ROUTES.TEMPLATE_DETAIL}?id=${id}&isLocal=${isLocal}`,
```

Comment:
this was a good removal, my opinion, since there's no need to set this up in a serializer ever if the ID is all we need for it. this was from having templates on their own page rather than just referenced from the group of templates in a modal view.